app/helpers/application_helper.rb

```diff
@@ -78,4 +78,7 @@ def disabled_class_for_no_selections(count)
       "disabled"
     end
   end
+  def from_advanced_search?
```

Comment:
I know it's a super simple helper method, but would you mind adding a unit test for this in `application_helper_spec.rb`?